next.js

```diff
@@ -0,0 +1,35 @@
+module.exports = {
+  extends: ["./react.js"],
+  overrides: [
+    {
+      files: [
+        "**/__tests__/next/**/*.test.tsx",
+        "**/__tests__/pages/**/*.test.tsx",
+        "**/__tests__/pages/**/*.test.jsx",
+      ],
+      rules: {
+        // off because of testing library
+        "jest/valid-expect": "off",
+        "jest/expect-expect": "off",
+      },
+    },
+  ],
+  rules: {
+    "import/no-internal-modules": [
+      "error",
+      {
+        allow: [
+          "**/*.css",
+          "@testing-library/**/*",
+          "react-dom/*",
+          "**/configuration/**/*",
+          "**/configuration/*",
+          "**/__fixtures__/**/*",
+          "**/__fixtures__/*",
```

Comment:
aren't these cypress-specific? they're not next-specific, right? maybe go for the cypress folder instead?

Reply:
True, moved to react setup that includes cypress.